webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -1,23 +1,24 @@
 <template>
-  <ds-button v-if="totalNotifications <= 0" class="notifications-menu" disabled icon="bell">
-    {{ totalNotifications }}
+  <ds-button v-if="!notificationsCount" class="notifications-menu" disabled icon="bell">
+    {{ unreadNotificationsCount }}
   </ds-button>
   <dropdown v-else class="notifications-menu" :placement="placement">
     <template slot="default" slot-scope="{ toggleMenu }">
-      <ds-button primary icon="bell" @click.prevent="toggleMenu">
-        {{ totalNotifications }}
+      <ds-button :primary="!!unreadNotificationsCount" icon="bell" @click.prevent="toggleMenu">
+        {{ unreadNotificationsCount }}
       </ds-button>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-20T21:01:09Z" title="Friday, September 20th 2019, 11:01:09 pm +02:00">Sep 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -35,7 +36,39 @@ export default {
   props: {
     placement: { type: String },
   },
+  created() {
+    setInterval(this.updateNotifications, REFRECH_MILISEC)
```


```suggestion
    setInterval(this.updateNotifications, REFRESH_MILLISEC)
```